xbmc/platform/darwin/tvos/input/LibInputTouch.mm

```diff
@@ -286,6 +267,7 @@ - (void)SiriDoubleSelectHandler:(UITapGestureRecognizer*)sender
 
 - (IBAction)IRRemoteUpArrowPressed:(UIGestureRecognizer*)sender
 {
+  CLog::Log(LOGDEBUG, "Input: IR remote up press (id: 1)");
```

Comment:
this line will be printed more than once (for different recogniser states). Either move it to the handler case or also print recogniser's state. (also in other places)